packages/databricks-vscode/resources/python/00-databricks-init.py

```diff
@@ -357,6 +359,99 @@ def df_html(df):
     html_formatter.for_type(SparkConnectDataframe, df_html)
     html_formatter.for_type(DataFrame, df_html)
 
+@logErrorAndContinue
+@disposable
+def register_spark_progress(spark):
+    try:
+        from pyspark.sql.connect.shell.progress import Progress
```

Comment:
```suggestion
        from pyspark.sql.connect.shell.progress import Progress as PysparkProgress
```

Reply:
We don't actually need the import anymore.